### PR TITLE
Feat react components

### PR DESCRIPTION
**Pull request by cjea**
purchase button + sell button + updatePrices action are functional

TODO: refactor to use AJAX calls for populating info rather than hardcoding it

### CODE REVIEW

File: client/Components/portfolio.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <input id="symbolInput" type="text" placeholder="Stock symbol . . ." />
+        <input id="numSharesInput" type="number" min="5" step="5" />
+        <button onClick={() => {
+          let buyOptions = {
+            shares: document.getElementById('numSharesInput').value,
+            stockSymbol: document.getElementById('symbolInput').value,
+            matchId: this.props.matchId,
+            userId: this.props.userId,
+            //this price is here just so it doesn't break right now; really this will come from async call
+            price: '111'
```
Comment (muddybarefeet) [inline client/Components/portfolio.jsx:32]: think on this

File: client/reducers/updatePricesReducer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function (state, action) {
+  return state.updateIn(['matches', '0', 'portfolio', 'stocks'], () => action.updatedStockArray);
+}
```
Comment (muddybarefeet) [inline client/reducers/updatePricesReducer.js]: new liner

File: client/test/test_helper.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+global.document = doc;
+global.window = win;
+
+Object.keys(window).forEach((key) => {
+ if (!(key in global)) {
+   global[key] = window[key];
+ }
+});
+
+chai.use(chaiImmutable);
```
Comment (muddybarefeet) [inline client/test/test_helper.js]: new liner

**Comment by cjea** [comment]
Closed to add new lines at ends of files

**Pull request closed without merging**